statusHandler/persister/persistentHandler.go

```diff
@@ -64,6 +65,7 @@ func (psh *PersistentStatusHandler) initMap() {
 	psh.persistentMetrics.Store(common.MetricTotalSupply, zeroString)
 	psh.persistentMetrics.Store(common.MetricTotalFees, zeroString)
 	psh.persistentMetrics.Store(common.MetricDevRewardsInEpoch, zeroString)
+	psh.persistentMetrics.Store(common.MetricGenesisBlockHash, emptyString)
```

Comment:
This is a bit misleading: we actually had 4 genesis block hashes
Where do we set this metric other than the empty string?